### DISCUSSION TITLE
Are unused ports in a component compiled?

### DISCUSSION THREAD

**Discussion by beavuhs** [nasa/fprime#2737] [Q&A]
If I do not hook up a component's port in the `topology.fpp` file, is that port compiled? Or do I need to make use of compiler flags to not compile specific ports?

**Comment by LeStarch** [comment]
If you are using a component, but not hooking up a specific port, then that port would still be compiled as it is required by the component.  Is there a specific reason that you do not wish the port to be compiled?
